src/InstallTool.php

```diff
@@ -193,7 +193,7 @@ public function hasOldDatabase()
 
         $column = $this->connection->fetchAssoc($sql." AND COLUMN_NAME = 'sections'");
 
-        return 'varchar(1022)' !== $column['Type'];
+        return 'blob NULL' !== $column['Type'];
```

Comment:
The documentation of this method says *“Checks if the database is older than version 3.2.”*, this would still be the case with the `in_array` check I think.